src/components/FilterDropdown.jsx

```diff
@@ -16,15 +18,18 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function FilterDropdown(props) {
-  const { label, items, defaultValue, removeNone, onChange } = props;
-  const [choice, setChoice] = useState(defaultValue);
+  const { label, items, value, removeNone, onChange } = props;
   const classes = useStyles();
+  const location = useLocation();
+  const params = queryString.parse(location.search, { parseNumbers: true });
```

Comment:
This component shouldn't be parsing the query string as it could be used for anything, including cases where the query string is not used.

Reply:
Ah ok I left it how it was when it merged from main, so should it be changed to `const params = queryString.parse(location.search);`?

Reply:
FilterDropdown should not be concerning itself over query parameters. It's just a visual component, it should not care what the URL is.

Reply:
Ok I see what you mean, I believe the current` FilterDropdown.jsx `file includes the `const params `anymore.

Reply:
Ok makes sense then I think the code below is what you mean
```javascript
if (!sortOptions.hasOwnProperty('deadline.asc')) {
    pruneQueryParam('sortBy');
}
```